test/gui/shared/scripts/helpers/ConfigHelper.py

```diff
@@ -100,6 +103,7 @@ def get_default_home_dir():
     'record_video_on_failure': False,
     'retrying': False,
     'video_recording_started': False,
+    'files_for_upload': CURRENT_DIR.parent.parent / 'upload-files',
```

Comment:
is this correct `parent.parent.parent`?
```suggestion
    'files_for_upload': CURRENT_DIR / 'upload-files',
```

Reply:
Used `parent.parent` to move up two levels from the `CURRENT_DIR`